src/app/_components/CardSpecialist/Organization/Card.js

```diff
@@ -16,6 +16,8 @@ import { CardButton } from '@components/CardSpecialist/CardButton';
 import { organizationPropType } from '@components/CardSpecialist/prop-types';
 
 export function CardOrganization({ organization, className, extended = false }) {
+  if (!organization) throw new Error('Organization is not found');
+
```

Comment:
@Yivgha my opinion is that PropTypes enough for such cases

Reply:
deleted